installation-bundle/src/Database/Installer.php

```diff
@@ -177,25 +178,28 @@ public function compileCommands(): void
                         switch (true) {
                             case 0 === strncmp($part, 'DROP ', 5):
                                 $return['ALTER_DROP'][md5($command)] = $command;
-                                $order[] = md5($command);
+                                $partsOrder[] = md5($command);
                                 break;
 
                             case 0 === strncmp($part, 'ADD ', 4):
                                 $return['ALTER_ADD'][md5($command)] = $command;
-                                $order[] = md5($command);
+                                $partsOrder[] = md5($command);
                                 break;
 
                             case 0 === strncmp($part, 'CHANGE ', 7):
                             case 0 === strncmp($part, 'RENAME ', 7):
                                 $return['ALTER_CHANGE'][md5($command)] = $command;
-                                $order[] = md5($command);
+                                $partsOrder[] = md5($command);
                                 break;
 
                             default:
                                 $parts[$i + 1] .= ','.$part;
                                 break;
                         }
                     }
+
+                    $order += array_reverse($partsOrder);
```

Comment:
I don’t think this works as expected, you need array_merge here IMO

Reply:
You are right. I had the associative array in mind, but this is numeric. :facepalm: 

Adjusted in add21b0067de89ded475117bb2407267bf8ca461.